fenix/app/src/main/java/org/mozilla/fenix/home/ActivityBreadcrumbs.kt

```diff
@@ -0,0 +1,25 @@
+package org.mozilla.fenix.home
+
+import android.app.Activity
+import androidx.navigation.NavDestination
+import org.mozilla.fenix.customtabs.ExternalAppBrowserActivity
+
+fun Activity.getBreadcrumbMessage(destination: NavDestination): String = when (this) {
```

Comment:
Should this functions move to `fenix/app/src/main/java/org/mozilla/fenix/ext/Activity.kt`?

Reply:
great suggestion @rocketsroger 🙂